doc/specs/#1571 - New Tab Menu Customization/#1571 - New Tab Menu Customization.md

```diff
@@ -76,11 +76,20 @@ There are five `type`s of objects in this menu:
   - The `"entries"` property specifies a list of menu entries that will appear
     nested under this entry. This can contain other `"type":"folder"` groups as
     well!
+  - The `"expand"` property accepts two values
+    - `auto`: When the folder only has one entry in it, don't actually create a
+      nested layer to then menu. Just place the single entry in the layer that
+      folder would occupy. (Useful for dynamic profile sources with only a
+      single entry).
+    - `always`: (**default**) Always create a nested entry, even for a single
+      sub-item.
   - The `allowEmpty` property will force this entry to show up in the menu, even
     if it doesn't have any profiles in it. This defaults to `false`, meaning
     that folders without any entries in them will just be ignored when
     generating the menu. This will be more useful with the `matchProfile` entry,
     below.

```

Comment:
Do we need to show a placeholder entry in the dropdown indicating that the folder is empty? Otherwise currently the dropdown is just a small rectangle that can easily be missed - we wouldn't want redundant bug reports since the menu is empty.